tests/Feature/PartnerManagementWorkflowTest.php

```
@@ -0,0 +1,260 @@
+
+            if (empty($partners)) {
+                // Partner not found - this is expected
+                $this->assertEmpty($partners);
+            } else {
+                $this->fail('Should not find non-existent partner');
+            }
+
+        } catch (ApiException $e) {
+            $this->fail('Should not throw exception: ' . $e->getMessage());
+        }
+    }
```

PHPMD (2.15.0)

The method it_handles_partner_not_found_scenario is not named in camelCase. (undefined)

(CamelCaseMethodName)